tests/Browser/CreateTaskCategoryTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\Models\User;
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use App\Models\TaskCategory;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+
+class CreateTaskCategoryTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+    /**
+     * A Dusk test example.
+     *
+     * @return void
+     */
+    public function testCreateCategory()
+    {
+        $taskCategry = TaskCategory::factory()->make();
+        $this->browse(function ($first) use ($taskCategry) {
+            $first->loginAs($taskCategry->user_id)
+                    ->visit('/taskCategory/create')
+                    ->assertTitle('Laravel')
+                    ->value('#name', $taskCategry->name)
+                    ->press('Add')
+                    ->assertPathIs('/taskCategory')
+                    ->assertSee('Data Added successfully!')
+                    ->assertSee($taskCategry->name);
+        });
+    }
```

Comment:
There needs to be test cases for negative flow i.e. assert validation errors on leaving name empty etc. Same for other forms/models as well. Always there should be multiple cases covering both positive and negative flows a user can take while performing that action.